decode_morse_message.rb

```diff
@@ -0,0 +1,27 @@
+def decode_char(char)
+  decoder = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+              '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+              '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+              '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----' => '1',
+              '..---' => '2', '...--' => '3', '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
+              '---..' => '8', '----.' => '9', '-----' => '0', ' ' => '', '/' => '/' }
+  decoder[char]
+end
+
```

Comment:
- [ ] As per the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md) of the project, "`Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").`", please kindly add a method that;
- ` splits input word by single space (char delimiter in Morse code)`,
- ` iterates through all chars from the input word`,
- ` reuses the `#decode_char` method to translate a single char symbol into a letter (DRY rule)`,
-` combines translated chars into a single word to be returned as the final output`